app/src/test/java/ch/epfl/sweng/swenggolf/DatabaseLocalTest.java

```diff
@@ -0,0 +1,5 @@
+package ch.epfl.sweng.swenggolf;
+
+public class DatabaseLocalTest {
+
+}
```

Comment:
Is there a current use case for this class to exist?

Reply:
This class is void like my life, I will delete it